database/migrations/sql/20180604_06_create_users.sql

```diff
@@ -12,13 +12,15 @@ CREATE TABLE gender (
 CREATE TABLE user_account (
   user_account_id                  SERIAL NOT NULL UNIQUE,
   user_name                        VARCHAR(100) NOT NULL,
-  user_secret                      VARCHAR(64) NOT NULL,
-  qr_code                          VARCHAR,
+  user_password                    VARCHAR(64),
+  qr_code                          VARCHAR UNIQUE,
   gender_id                        INT NOT NULL,
-  email                            VARCHAR(100) NOT NULL UNIQUE,
+  email                            VARCHAR(100) UNIQUE,
   phone_number                     VARCHAR(20),
   post_code                        VARCHAR(10),
   birth_year                       INT,
+  is_email_confirmed               BOOLEAN NOT NULL DEFAULT false,
+  is_phone_number_confirmed        BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Are we going to be sending text message confirmations?

Reply:
Maybe. Either way, we should know whether we know this, if you get me.

Reply:
Nope, but it's not overly important right now.